src/components/scenes/DevTestScene.tsx

```diff
@@ -113,7 +113,7 @@ export function DevTestScene(props: Props) {
   let balance = coreWallet?.balanceMap.get(tokenId) ?? ''
   if (eq(balance, '0')) balance = ''
   const headerText = 'Select Wallet'
-  const headerCallback = () => console.log('Header pressed')
+  const headerCallback = async () => console.log('Header pressed')
```

Comment:
This `async` doesn't do anything, because there is nothing to `await`. Maybe the type below should be `headerCallback?: () => (void | Promise<void>)`.

That's how most of our tappable components work - if you return a promise, we handle it, and if you return void, we handle that too.